charts/cp-schema-registry/values.yaml

```diff
@@ -83,6 +83,39 @@ resources: {}
 ## Custom pod annotations
 podAnnotations:
   cluster-autoscaler.kubernetes.io/safe-to-evict: "true"  # Allow the autoscaler to downscale the cluster.
+  sidecar.istio.io/inject: "false"
+  ad.datadoghq.com/cp-schema-registry-server.logs: '[{"source":"schema-registry","service":"{{ .Release.Name }}"}]'
+  ad.datadoghq.com/cp-schema-registry-server.tags: '{}'
+  ad.datadoghq.com/cp-schema-registry-server.check_names: '["openmetrics"]'
+  ad.datadoghq.com/cp-schema-registry-server.init_configs: '[{}]'
+  ad.datadoghq.com/cp-schema-registry-server.instances: |
+    [{
+    "prometheus_url": "http://%%host%%:5556/metrics",
+    "namespace": "schema_registry",
+    "metrics": [
+      "cp_kafka_schema_registry_jetty_metrics_connections_active",
+      "cp_kafka_schema_registry_jetty_metrics_connections_opened_rate",
+      "cp_kafka_schema_registry_jetty_metrics_connections_closed_rate",
+      "cp_kafka_schema_registry_jersey_metrics_request_error_rate",
+      "cp_kafka_schema_registry_master_slave_role",
+      "cp_kafka_schema_registry_jersey_metrics_request_byte_rate",
+      "cp_kafka_schema_registry_jersey_metrics_request_latency_avg",
+      "cp_kafka_schema_registry_jersey_metrics_request_size_avg",
+      "jvm*"
+    ],
+    "type_overrides": {
+      "*": "gauge",
+      ".+_total": "counter"
+    }
+    }]
+  
+
+## Custom pod labels
+podLabels:
+  team: "team_dbre"
+  tags.datadoghq.com/service: "{{ .Release.Name }}"
```

Comment:
To reduce the length of the values file I think it would be ok to set this directly in the deployment

Reply:
Shouldn't we do like in other charts ? (cf the other comment about that)
So: put the labels in values.yaml and in _helpers.yml.

Reply:
should be ok now